src/pdf/DriverOverview/DriverOverviewPdf.tsx

```diff
@@ -203,8 +230,20 @@ const DriverOverviewCard: React.FC<DriverOverviewCardProps> = ({ data }) => {
                         COMPLETION OF DELIVERIES
                     </Text>
                 </View>
-                <View style={[styles.flexColumn, { width: "100%" }]}>{header}</View>
-                <View style={[styles.tableSection, styles.flexColumn]}>{table}</View>
+                <View style={[styles.h2text, styles.collectionOrDeliveryHeader, styles.flexRow]}>
+                    <Text style={styles.collectionOrDeliveryHeader}>Deliveries</Text>
+                    <FontAwesomeIconPdfComponent faIcon={faTruck}></FontAwesomeIconPdfComponent>
+                </View>
+                <View style={[styles.flexColumn, { width: "100%" }]}>{deliveriesHeader}</View>
+                <View style={[styles.tableSection, styles.flexColumn]}>{deliveries}</View>
+                <View style={[styles.h2text, styles.collectionOrDeliveryHeader, styles.flexRow]}>
+                    <Text style={[styles.collectionOrDeliveryHeader]}>Collections</Text>
+                    <FontAwesomeIconPdfComponent
+                        faIcon={faShoePrints}
+                    ></FontAwesomeIconPdfComponent>
+                </View>
```

Comment:
Think I like the idea of saying "No deliveries/collections" instead of an empty table if the array is empty (or maybe not even showing the section at all)